docs/generated/errors-list.md

```diff
@@ -76,4 +76,8 @@ The `galasactl` tool can generate the following errors:
 - GAL1073E: Could not get run artifacts. Reason: '{}'
 - GAL1074E: Could not download artifact '{}'. Reason: '{}'
 - GAL1075E: run name '{}' is invalid. Expected it to be in format starting with letters, and ending in a number with no non-alphanumeric characters.
+- GAL1076E: Badly formed from or to value '{}' specified in the age parameter. The value could not be converted into an integer value.
+- GAL1077E: Invalid value '{}' detected for age parameter. The from value must be older than the to value.
+- GAL1079E: Badly formed age parameter '{}' specified. Age of the tests should be specified in FROM:TO or FROM format with the units w (weeks), d (days) or h (hours).
```

Comment:
add `where the FROM or TO parts of the value are integers, separated by a ':' separator.